paddle/fluid/inference/api/mkldnn_quantizer_config.cc

```diff
@@ -35,6 +35,9 @@ MkldnnQuantizerConfig::MkldnnQuantizerConfig() {
   rules_["pool2d"]["X"] = ScaleAlgo::KL;
   rules_["pool2d"]["Out"] = ScaleAlgo::KL;
 
+  rules_["softmax"]["X"] = ScaleAlgo::KL;
+  rules_["softmax"]["Out"] = ScaleAlgo::NONE;
```

Comment:
I think the scale should be calculated or if it is not calculated as written now, it should be set in other place as we discussed to i.e. 126.